### PR TITLE
Kernel secret-izing and updating README.md

### PR DESCRIPTION
**Pull request by evanjamesjackson**

### CODE REVIEW

**Review by lukemacausland — COMMENTED** [review]
@evanjamesjackson - Do we want the `composer.lock`? It was never committed before which is why I never committed it in my past PR.

**Comment by evanjamesjackson** [comment]
> @evanjamesjackson - Do we want the `composer.lock`? It was never committed before which is why I never committed it in my past PR.

Good catch, we definitely don't

File: .docker/Dockerfile [hunk 1]
```diff
@@ -12,8 +12,9 @@ COPY --chown=www-data:www-data /composer.* /var/www/html
 COPY /.docker/composer-install-dependencies.sh /composer-install-dependencies.sh
 RUN /composer-install-dependencies.sh
 COPY --chown=www-data:www-data / /var/www/html
-RUN set -eux; \
-	cd /var/www/html; \
+RUN --mount=type=secret,id=kernel-secret,uid=1000 \
+    set -eux; \
```
Comment (lukemacausland) [inline .docker/Dockerfile:16]: Do we need this line if its already set on line 2?

```suggestion
```

Comment (lukemacausland) [reply .docker/Dockerfile:16]: Or does it need to be specified per `RUN` command?

Comment (evanjamesjackson) [reply .docker/Dockerfile:16]: It is required per RUN

**Review by IronSean — APPROVED** [review]

**Pull request merged**